flow/src/org/labkey/flow/webparts/AnalysesWebPart.java

```diff
@@ -76,7 +76,7 @@ protected void populateButtonBar(DataView view, ButtonBar bar)
             }
 
             ActionURL createRunGroupURL = PageFlowUtil.urlProvider(ExperimentUrls.class).getCreateRunGroupURL(getContainer(), getReturnURL(), false);
-            ActionButton createExperiment = new ActionButton(createRunGroupURL, "Create Analysis Folder");
+            ActionButton createExperiment = new ActionButton(createRunGroupURL, "Create Analysis Folder", ActionButton.Action.POST);

```

Comment:
Does this change belong here or on the CSRF branch?

Reply:
Yup it sure does belong on CSRF